backend/src/utilities/crud_factory.ts

```diff
@@ -21,10 +21,10 @@ function create<T extends IMongo_Entity>(model:Model<T>,
         (req: Request, res: Response) => {
             let new_entity: T = req.body;
             model.create(new_entity)
-                .then((nent:T) => res.status(201).send(
+                .then((new_ent) => res.status(201).send(
```

Comment:
is `new_ent` not a `T`?

Reply:
Yes it is, but not always. often when we are creating a new entity, it will not have all of that types required fields (i.e. `_id`) so if we send it one without the required fields, we get a type error. that is no good, so we hurt ourselves by specifying a type here.